pkg/controller/networkpolicy/validate.go

```diff
@@ -213,6 +238,22 @@ func (v *NetworkPolicyValidator) Validate(ar *admv1.AdmissionReview) *admv1.Admi
 			}
 		}
 		msg, allowed = v.validateAntreaPolicy(&curANP, &oldANP, op, ui)
+	case "Service":
+		klog.V(2).Info("Validating K8s Service")
```

Comment:
Use structured logging for all the klog calls that you add or modify with this PR

Reply:
Done.